test/validate/ferc1_test.py

```diff
@@ -65,16 +63,16 @@ def test_no_null_cols_ferc1(pudl_out_ferc1, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,expected_rows",
     [
-        ("fbp_ferc1", 20_573),
-        ("fuel_ferc1", 31_265),
-        ("plant_in_service_ferc1", 26_933),
-        ("plants_all_ferc1", 51_456),
-        ("plants_hydro_ferc1", 6_782),
-        ("plants_pumped_storage_ferc1", 710),
-        ("plants_small_ferc1", 15_584),
-        ("plants_steam_ferc1", 28_380),
-        ("pu_ferc1", 6_946),
-        ("purchased_power_ferc1", 190_228),
+        ("fbp_ferc1", 25_097),
+        ("fuel_ferc1", 48_218),
+        ("plant_in_service_ferc1", 311_794),
+        ("plants_all_ferc1", 54_276),
+        ("plants_hydro_ferc1", 6_796),
+        ("plants_pumped_storage_ferc1", 544),
```

Comment:
this is the only one that actually went down. i believe this is because we are actually dropping rows now!